cli/npm/installer/common/lifecycle_scripts.rs

```diff
@@ -37,37 +70,6 @@ pub trait LifecycleScriptsStrategy {
   fn has_warned(&self, package: &NpmResolutionPackage) -> bool;
 
   fn has_run(&self, package: &NpmResolutionPackage) -> bool;
-
-  fn did_run_scripts(
```

Comment:
I got rid of the code that allowed for possibly executing scripts in the global dir. We can add it back later and have it use a life cycle script executor if we need it in the future.